frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -886,10 +902,9 @@ export default function SurveyEdit(): JSX.Element {
             <LemonDivider vertical />
             <div className="flex flex-col items-center h-full w-full sticky top-0 pt-8" style={{ maxWidth: 320 }}>
                 <SurveyFormAppearance
-                    activePreview={activePreview}
+                    activePreview={selectedQuestion || 0}
```

Comment:
Why do we have to OR this to 0 if its default value is already 0 on the logic? :0